ui/cap-react/src/antd/forms/templates/ArrayFieldTemplates/NormalArrayFieldTemplate.js

```diff
@@ -300,16 +296,10 @@ const NormalArrayFieldTemplate = ({
             />
           </Col>
         )}
-
-        {(uiSchema["ui:description"] || schema.description) && (
-          <Col span={24} style={{ ...DESCRIPTION_COL_STYLE, padding: "0" }}>
-            <DescriptionField
-              description={uiSchema["ui:description"] || schema.description}
-              id={`${idSchema.$id}__description`}
-              key={`array-field-description-${idSchema.$id}`}
-            />
-          </Col>
```

Comment:
if we don't keep `DescriptionField` we should pass `key`, `id` to `FieldHeader`, so we dont break any features from rjsf